projects/distributed-tracing/src/shared/dashboard/widgets/trace-detail/data/trace-detail-data-source.model.ts

```diff
@@ -57,6 +57,8 @@ export class TraceDetailDataSourceModel extends GraphQlDataSourceModel<TraceDeta
   protected constructTraceDetailData(trace: Trace): TraceDetailData {
     return {
       id: trace[traceIdKey],
+      traceId: trace.traceId as string, // The traceId is real Trace ID. NOT Symbol('traceId').
```

Comment:
Can we replace or remove `id` then? I think it's confusing to have `id`, `traceId` and `entrySpanId` - there's only two distinct values, right?